vllm/model_executor/layers/quantization/utils/marlin_utils_fp8.py

```diff
@@ -310,26 +305,18 @@ def prepare_moe_fp8_layer_for_marlin(
         scales = torch.cat([x.unsqueeze(0) for x in tensor_list], 0)
         if input_dtype != torch.float8_e4m3fn:
             scales = fp8_fused_exponent_bias_into_scales(scales)
-        scales = torch.nn.Parameter(scales, requires_grad=False)
-
-        setattr(layer, name + "_weight_scale", scales)
+        return scales
 
-    # BIAS
```

Comment:
none of the quant methods which use this function support bias

Reply:
maybe we can add it back in another PR. the reason I removed it is that its hard to test